Intersect.Server/Localization/Strings.cs

```diff
@@ -416,6 +416,12 @@ public sealed class CommandoutputNamespace : LocaleNamespace
 
             [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]            public readonly LocalizedString cpsunlocked = @"CPS Unlocked";
 
+            [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]            public readonly LocalizedString variablechanged = @"Variable {00} has been set to {01} from {02}";
+
+            [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]            public readonly LocalizedString variablenotfound = @"Variable {00} is not a valid server variable GUID";
+
+            [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]            public readonly LocalizedString variableexception = @"Variable {00} threw exception while attempting to modify: {00}";
+
```

Comment:
Proper case names, i.e. `VariableChanged`

`VariableNotFound` -> `VariableNotFoundById`

`VariableException` isn't needed (because of my review of the first file)

Reply:
Done